client/web/web.go

```diff
@@ -77,25 +81,26 @@ type browserSession struct {
 	// ID is the unique identifier for the session.
 	// It is passed in the user's "TS-Web-Session" browser cookie.
 	ID            string
-	SrcNode       tailcfg.StableNodeID
+	SrcNode       tailcfg.NodeID
```

Comment:
@willnorris auth paths use the `NodeID`, thus the change here 😵 

could include both if we want, although since this is in-memory, we probably are okay - migrations are easier down the line